app/(navigation)/(code)/util/formatCode.ts

```diff
@@ -9,10 +9,25 @@ const parsers = {
   CSS: { import: () => import("prettier/plugins/postcss"), name: "css" },
   SCSS: { import: () => import("prettier/plugins/postcss"), name: "css" },
   YAML: { import: () => import("prettier/plugins/yaml"), name: "yaml" },
+  Python: { import: () => Promise.resolve({ default: {} }), name: "python" },
 };
 
 export const formatterSupportedLanguages: Language["name"][] = Object.keys(parsers);
 
+const ruffConfig = {
+  preview: false,
+  builtins: [],
+  "target-version": "py312",
+  "line-length": 80,
+  "indent-width": 4,
+  format: {
+    "indent-style": "space",
+    "quote-style": "double",
+    "skip-magic-trailing-comma": false,
+    "line-ending": "auto",
+  },
+};
```

Comment:
You can use `Workspace.defaultSettings` to get the default settings

Reply:
Thank you for the suggestion @MichaReiser! Addressed.